src/assets/translations/pt/main.json

```diff
@@ -146,11 +152,16 @@
           "validator": "O Validador é responsável por processar transações, armazenar dados e adicionar blocos ao blockchain.",
           "gasFees": "As taxas de gás em %{networkName} são muito mais baixas do que as da Ethereum Mainnet. Esta taxa é paga diretamente aos validadores que protegem e operam a rede e não é paga à ShapeShift."
         }
+      },
+      "foxyOverview": {
+        "header": "Deposit FOX to get FOXy to earn"
```

Comment:
This is english.

Reply:
@thesmithdao requesting another look at the locale file and new pull request 🙏